### ISSUE TITLE
fury warrior t30 doesn't work

### ISSUE THREAD

**Issue by machajr** [Hekili/hekili#2596]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

action.bloodthirst.crit_pct_current >= 95 this doesnt look working i think its only check your crit rate not your buffs for bloodthirst there is a weakaura for that that may help you https://wago.io/PKtmE426S

### How to Reproduce

action.bloodthirst.crit_pct_current >= 95 this doesnt look working i think its only check your crit rate not your buffs for bloodthirst there is a weakaura for that that may help you https://wago.io/PKtmE426S

### Snapshot (Link)

https://pastebin.com/5vMSTc9Y

**Comment by c2-gh** [comment]
I have also observed this issue. To be clear, this results in Bloodthirst not ever being recommended regardless of how many 4pc stacks you currently have. Unless Raging Blow/Onslaught are both on cooldown, which is almost never the case with the standard T30 build.

**Comment by BigBong69** [comment]
Would also like to point out that recklessness is NOT updating the value of crit.pct.current. Both images have the same crit% even though reck is up for one of them (20% crit)

![image](https://github.com/Hekili/hekili/assets/78175329/179e435d-153f-4ee8-aef1-cbd3d10ee452)

![image](https://github.com/Hekili/hekili/assets/78175329/08606595-1e1a-4a65-a334-83ff775d25fb)

**Comment by c2-gh** [comment]
The issue seems to persist with the latest version (v10.1.0-1.0.0k) unfortunately.

**Comment by Hekili** [comment]
Duplicate of #2600; resolved in `v10.1.0-1.0.0l`.

**Comment by machajr** [comment]
its still doesnt recommend bloodthirst.

https://pastebin.com/KpxeppuD

**Comment by Hekili** [comment]
@machajr Take your snapshot at a time when you think it should be recommending Bloodthirst but isn't.  You provided an out of combat snapshot when your precombat usage of Avatar would occur.

**Comment by machajr** [comment]
https://pastebin.com/V502c3JR

**Comment by machajr** [comment]
ignore that one i ll post new one

**Comment by machajr** [comment]
https://pastebin.com/q1BkPcRd

**Comment by Hekili** [comment]
@machajr That snapshot still isn't far enough into the fight -- Ravager is recommended, which is higher priority than Bloodthirst/Bloodbath even with T30 buffs increasing your crit.

What is your base crit %?

In the snapshot, you have Recklessness up (+20% crit) and 5 stacks of Merciless Assault (+5 x 10% crit) for a total 70% increased critical chance.  If your base crit is less than 25%, you're not going to reach the 95% crit threshold for these entries for Bloodthirst/Bloodbath:

```
actions.single_target+=/bloodbath,if=set_bonus.tier30_4pc&action.bloodthirst.crit_pct_current>=95
actions.single_target+=/bloodthirst,if=set_bonus.tier30_4pc&action.bloodthirst.crit_pct_current>=95
```

You should still see Bloodbath and Bloodthirst recommended in the non-tier related circumstances.

Can you link a sim of your character here?
* https://www.raidbots.com/simbot/quick

This will let me confirm whether the addon and your sim are currently matched.

**Comment by machajr** [comment]
https://www.raidbots.com/simbot/report/9TzYB9FjeJr3ybu8jZZNPF

**Comment by Hekili** [comment]
Actual fix will be in `1.0.1`.
